src/ui/server/controllers/insurance/business/company-details/index.ts

```diff
@@ -5,17 +5,22 @@ import corePageVariables from '../../../../helpers/page-variables/core/insurance
 import api from '../../../../api';
 import companiesHouseValidation from './validation/companies-house';
 import companyHouseResponseValidation from './validation/companies-house-response';
+import companyDetailsValidation from './validation/company-details';
 
 import { companyHouseSummaryList } from '../../../../helpers/summary-lists/company-house-summary-list';
 
-const { COMPANY_HOUSE } = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+const { EXPORTER_BUSINESS } = FIELD_IDS.INSURANCE;
+const { COMPANY_HOUSE, YOUR_COMPANY } = EXPORTER_BUSINESS;
+const { TRADING_NAME } = YOUR_COMPANY;
 const { COMPANY_DETAILS } = PAGES.INSURANCE.EXPORTER_BUSINESS;
 const { COMPANY_DETAILS: companyDetailsTemplate } = TEMPLATES.INSURANCE.EXPORTER_BUSINESS;
 const { COMPANY_HOUSE_SEARCH } = ROUTES.INSURANCE.EXPORTER_BUSINESS;
+const { COMPANY_DETAILS: COMPANY_DETAILS_ROUTE } = ROUTES.INSURANCE.EXPORTER_BUSINESS;
 
 const PAGE_VARIABLES = {
-  POST_ROUTE: COMPANY_HOUSE_SEARCH,
-  FIELDS: COMPANY_HOUSE,
+  COMPANIES_HOUSE_POST_ROUTE: COMPANY_HOUSE_SEARCH,
+  COMPANY_DETAILS_POST_ROUTE: COMPANY_DETAILS_ROUTE,
+  FIELDS: EXPORTER_BUSINESS,
```

Comment:
What do you think about having it like this?

```js
{
  ...
  POST_ROUTES: {
    COMPANIES_HOUSE: 'abc',
    COMPANY_DETAILS: 'abc',
  }
}
```

Reply:
done